docs/indexer/build/subgraph-style.md

```diff
@@ -0,0 +1,239 @@
+| notLikeInsensitive             | - | `field: { notLikeInsensitive: value }` |
+| equalToInsensitive             | - | `field: { equalToInsensitive: value }` |
+| notEqualToInsensitive          | - | `field: { notEqualToInsensitive: value }` |
+| distinctFromInsensitive        | - | `field: { distinctFromInsensitive: value }` |
+| notDistinctFromInsensitive     | - | `field: { notDistinctFromInsensitive: value }` |
+| inInsensitive                  | - | `field: { inInsensitive: [value1, value2] }` |
+| notInInsensitive               | - | `field: { notInInsensitive: [value1, value2] }` |
+| lessThanInsensitive            | - | `field: { lessThanInsensitive: value }` |
+| lessThanOrEqualToInsensitive   | - | `field: { lessThanOrEqualToInsensitive: value }` |
+| greaterThanInsensitive         | - | `field: { greaterThanInsensitive: value }` |
+| greaterThanOrEqualToInsensitive| - | `field: { greaterThanOrEqualToInsensitive: value }` |
+
+
+Example:
+```graphql
+# subquery style
+query Accounts {
+    accounts(
+        first: 1
+        filter: {
+            id: {
+                in: ["12idzncwtgfd1nyayku6av5nmmm9i6uj3gmqemww1uf6gtnt"]
+            }
+        }
+    ) {
+        nodes {
+            id
+            firstTransferBlock
+            lastTransferBlock
+        }
+        totalCount
+    }
+}
+```
+
+The subgraph uses where as the query field, and the filter condition looks like this: `<field>_<operators>`.
+```graphql
+# subgraph style
+query Accounts {
+    accounts(
+        first: 1
+        where: {
+            id_in:  ["12idzncwtgfd1nyayku6av5nmmm9i6uj3gmqemww1uf6gtnt"]
+        }
+    ) {
+        id
+        firstTransferBlock
+        lastTransferBlock
+    }
+}
+```
+
+## List sort
+The `subquery` only requires the `orderBy` field, while the `subgraph` requires two fields: `orderBy` and `orderDirection`.
+
+Example:
+```graphql
+# subquery style
+query Accounts {
+    accounts(
+        first: 1
+        orderBy: ID_ASC
+    ) {
+        nodes {
+            id
+            firstTransferBlock
+            lastTransferBlock
+        }
+        totalCount
+    }
+}
+```
+
+```graphql
+# subgraph style
+query Accounts {
+    accounts(
+        first: 1
+        orderBy: id
+        orderDirection: asc
+    ) {
+        id
+        firstTransferBlock
+        lastTransferBlock
+    }
+}
+```
+
+## skip item
+The `subquery` passes the `offset` field, while the `subgraph` uses the `skip` field.
+
+Example:
+```graphql
+# subquery style
+query Accounts {
+    accounts(
+        first: 1
+        offset: 10
+    ) {
+        nodes {
+            id
+            firstTransferBlock
+            lastTransferBlock
+        }
+        totalCount
+    }
+}
+```
+
+```graphql
+# subgraph style
+query Accounts {
+    accounts(
+        first: 1
+        skip: 10
+    ) {
+        id
+        firstTransferBlock
+        lastTransferBlock
+    }
+}
+```
+
+## Other unsupported features
```

Comment:
Can you please explain the differences between the graphs query service and this query service. What happens with different primitive types. e.g. BigDecimal

Also differences with metadata